blog/2024-09-24-python313.md

```diff
@@ -0,0 +1,56 @@
+---
+authors:
+  - isuruf
+tags: [infrastructure]
+---
+
+# Python 3.13 Release candidate builds on conda-forge
+
+conda-forge now supports python 3.13 release candidates on conda.
+You can create a new environment with python 3.13 by running the
+command:
+
+    conda create -n py313 python=3.13 -c conda-forge/label/python_rc -c conda-forge
```

Comment:
```suggestion
    conda create -n py313 python=3.13 --override-channels -c conda-forge/label/python_rc -c conda-forge
```

Do you think we should emphasize this for folks mixing channels?

Reply:
We really should make strict channel priority the default in conda. When creating environments, I don't think this changes anything.

Reply:
Noted: https://github.com/conda/conda/issues/14274